source/documentation/other-topics/migrate-to-live.html.md.erb

```diff
@@ -266,6 +266,13 @@ Earlier on in this migration you will have [migrated the ServiceAccount module](
 
 An example of GitHub Actions config is provided in this repo: https://github.com/ministryofjustice/cloud-platform-reference-app-github-action. It deploys the app to both the live-1 and live clusters.
 
+### Namespace dependencies
+
+If you have any terraform code referencing to secrets of your namespace mentioned in other namespaces in the ["live-1" directory][live-1-folder]
```

Comment:
This could do with being clearer. Maybe describe how the problem will show up, if they don't do this step?

Reply:
@davidread , I have added additional note on this. Does this read ok?